source/form.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Пинк</title>
+</head>
+
+<body class="page-body">
+  <header class="page-header">
+    <nav class="main-nav">
+      <a href="index.html" class="main-nav__logo">
+        <img src="./img/pink-logo-mobile.svg" width="74" height="23" alt="Логотип сайта Пинк">
+      </a>
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list">
+          <li class="main-nav__item">
+            <a href="index.html">Главная</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="catalog.html">Фотография</a>
+          </li>
+          <li class="main-nav__item">
+            <a>Конкурс</a>
+          </li>
+          <li class="main-nav__item">
+            <a>Html academy</a>
+          </li>
+        </ul>
+      </div>
+      <div class="intro">
+        <h1 class="intro__title">Конкурс на самое безбашенное путешествие с Пинк</h1>
+      </div>
+    </nav>
+  </header>
+  <main class="main-page">
+    <div class="promo">
+      <div class="promo__wrapper">
+        <p class="promo__text">Поделитесь своей историей и получите шанс выиграть ценный приз — 1000 миль на вашу
+          бонусную карту!</p>
+      </div>
+    </div>
+    <form class="main-form" action="https://echo.htmlacademy.ru" method="get">
```

Comment:
```suggestion
    <form class="main-form" action="https://echo.htmlacademy.ru" method="post">
```
Метод `GET` передаёт данные в виде параметров в URL. Здесь такой вариант не подходит.